### DISCUSSION TITLE
Squash and Merge input text should match PR title

### DISCUSSION THREAD

**Discussion by parthea** [community/community#7417] [Pull Requests]
Hello,

When I edit the PR title, I expect the new edited title to appear when I click the `Squash and merge` button but instead I see the original PR title.

Steps to reproduce:
1. Create a branch off the default branch
2. Push a commit to a newly created branch with title `Title 1`
3. Open a PR, you'll see `Title 1` as the PR title
4. Rename the PR title in Github to `Title 2`
5. Click the `Squash and Merge` button
6. You'll see `Title 1` instead of `Title 2`

**Comment by parthea** [comment]
Oops... Duplicate of https://github.com/github/feedback/discussions/6102

**Reply by KlausEverWalkingDev** [reply]
You could mark this answer as the right answer for your post :)
